x-pack/plugins/security_solution/common/utils/path_placeholder.ts

```diff
@@ -7,7 +7,7 @@
 
 import { ConditionEntryField, OperatingSystem, TrustedAppEntryTypes } from '../endpoint/types';
 
-export const placeholderText = {
+export const placeholderText = () => ({
```

Comment:
nit: As this function is returning a value it should starts with `get`, for example: `getPlaceholderText()`

Reply:
done ab49010e404be6dc2955102a7608ec2bb29a5413